docs/how-to-guides/customise-multipass/set-up-a-graphical-interface.md

```diff
@@ -16,7 +21,21 @@ You can display the graphical desktop in various ways. In this document, we desc
 
 The images used by Multipass do not come with a graphical desktop installed. For this reason, you will have to install a desktop environment (here we use `ubuntu-desktop` but there are as many other options as flavours of Ubuntu exist) along with the RDP server (we will use `xrdp` but there are also other options such as `freerdp`).
 
-To do this, first you need to log into a running Multipass instance. Start by listing your instances:
+```{admonition} Prerequisites for ubuntu-desktop
+:class: important
+
+Installing a desktop environment requires more resources than the default Multipass instance settings. To successfully set up and run a graphical interface, you must have an instance with at least **10GB** of disk space and **2GB** of RAM.
```

Comment:
I'm definitely not a technical writer, so don't consider this an expert opinion, but would be be better to say "it it recommended to have an instance with at least..." rather than "you must have an instance..."?

To me, using "must have" means that we are following a specific requirement, when in actuality it is just the next highest, round, clean number for setting system resources for an instance.

Reply:
In addition to what @sharder996 mentioned, it may also be worth reiterating that these requirements/recommendations only apply to `ubuntu-desktop`? (Other distros might have higher or lower requirements.) Maybe something like the following:

> Installing a desktop environment requires more resources than the default Multipass instance settings. To successfully set up and run a graphical interface under `ubuntu-desktop`, you will need an instance with at least **10GB** of disk space and **2GB** of RAM.